cfgov/v1/jinja2/v1/layouts/base.html

```diff
@@ -60,7 +60,7 @@
             {%- endblock -%}
           ">
     <meta name="searchgov_custom1" content="{{ language }}">
-    <meta name="searchgov_custom2" content="{{ self.desc() }}">
+    <meta name="searchgov_custom2" content="{{ encode_b32_string(self.desc()) }}">
```

Comment:
How about a comment here to document why we are doing this?

```suggestion
    {# Base32 encoding used to avoid an issue with Search.gov custom fields losing capitalization. -#} 
    <meta name="searchgov_custom2" content="{{ encode_b32_string(self.desc()) }}">
```